src/ShopListFilterBundle/objects/db/TPkgShopListfilterItem.class.php

```diff
@@ -134,6 +148,10 @@ public function GetURLInputName()
      */
     public function GetActiveSettingAsHiddenInputField()
     {
+        /**
+         * @psalm-suppress InvalidArgument
+         * @FIXME Passing an array (`aActiveFilterData`) to `TGlobal::OutHTML` which expects a string. Due to `htmlentities` being used in that method, this will return an empty string up to PHP7.4 but result in an fatal error in PHP8+
```

Comment:
This method will fail in PHP8+. I have not yet checked the 7.1.x branch to see if this portion of the code has changed though

Reply:
I doublechecked on the `7.1.x` branch and the code hasn't changed there. This is a PHP8 incompatibility that currently exists in the branch that should be supporting 8.0